app/src/main/java/net/gsantner/markor/activity/MainActivity.java

```diff
@@ -71,13 +71,18 @@
 
     private boolean _doubleBackToExitPressedOnce;
 
+
```

Comment:
can be removed

Reply:
Should I make another PR with the necessary changes?

Reply:
no, just reset those lines and make commit :)